packages/legend-manual-tests/src/__tests__/roundtrip-grammar/cases/DSLMastery-basic.pure

```diff
@@ -0,0 +1,41 @@
+###Pure
+Class org::legend::Person
+{
+  name: String[0..1];
+}
+
+###Mapping
+Mapping org::legend::PersonMapping
+(
+)
+
+###Service
+Service org::legend::PersonService
+{
+  pattern: 'person';
+  documentation: 'Person test service';
+  autoActivateUpdates: true;
+  execution: Single
+  {
+    query: src: org::legend::Person[1]|$src.name;
+    mapping: org::legend::PersonMapping;
+    runtime:
+    #{
+      mappings:
+      [
+        org::legend::PersonMapping
+      ];
+    }#;
+  }
+  testSuites:
+  [
+
+  ]
+  test: Single
+  {
+    data: 'test';
+    asserts:
+    [
+    ];
+  }
```

Comment:
There will be many changes to the Mastery grammar in the coming weeks, so I did not want to include a round trip test here yet to prevent breaking changes to studio build as the Mastery grammar changes.

Leaving this placeholder test file so that we can add the roundtrip test once the Mastery grammar is in a stable condition